src/Stryker.Core/Stryker.Core/Initialisation/InitialTestProcess.cs

```diff
@@ -4,6 +4,7 @@
 using Stryker.Core.TestRunners;
 using System;
 using System.Diagnostics;
+using Stryker.Core.Testing;
```

Comment:
It is used, cannot be removed.

Reply:
using Stryker.Core.Testing was added to be able to use Chalk. Since Chalk is no longer used, this using is not neccesary. But I won't block this PR just on some unused usings, it doesn't matter that much.